qcfractal/tests/test_storage.py

```diff
@@ -1258,21 +1241,21 @@ def test_collections_projection(storage_socket):
     # print('All: ', ret["data"])
 
     projection = {"records", "name"}
-    ret = storage_socket.get_collections(collection=collection, name=name, include=projection)
+    ret = storage_socket.get_collections(collection=collection, name=name, projection=projection)
```

Comment:
Where did we end up on the `include/projection` here?

Reply:
It was my understanding that we were going to rename projection to include. This is the subject of PR #457.

Reply:
In this PR it's still projection, but as Matt mentioned, it will be renamed in his PR.